tests/integration/test_labeling_parameter_overrides.py

```diff
@@ -33,4 +34,34 @@ def test_labeling_parameter_overrides(project, rand_gen):
     # currently this doesn't work so the count remains 3
     assert len(list(project.labeling_parameter_overrides())) == 1
 
+    with pytest.raises(TypeError) as exc_info:
+        data = [(data_rows[12], "a_string", 3)]
+        project.set_labeling_parameter_overrides(data)
+    assert str(exc_info.value) == \
+        f"Priority must be an int. Found <class 'str'> for data_row {data_rows[12]}. Index: 0"
```

Comment:
Is it better to show the use the type of the value instead of the value?

Like <class 'str'> instead of "dog"

Reply:
We are showing both. This is just the test so the type is hard coded.